src/mozanalysis/experiment.py

```diff
@@ -1152,13 +1152,87 @@ def get(self, bq_context, analysis_window):
             self._build_analysis_window_subset_query(analysis_window)
         ).to_dataframe()
 
+    def get_full_data(self, bq_context):
+        """Get the full DataFrame from TimeSeriesResult.
+
+        This DataFrame has a row for each client for each period of the time
+        series and may be very large. A warning will print the size of data
+        to be downloaded.
+
+        Args:
+            bq_context (BigQueryContext)
+        """
+        size = self._get_table_size(bq_context)
+
+        print(
+            "Downloading {full_table_name} ({size} GB)".format(
+                full_table_name=self.fully_qualified_table_name,
+                size=size
+            )
+        )
+
+        return bq_context.run_query(
+            """
+            SELECT * EXCEPT (client_id)
+            FROM {full_table_name}
+            """.format(
+                full_table_name=self.fully_qualified_table_name
+            )
+        ).to_dataframe()
```

Comment:
This could also be written as something like, if it is fine to keep the `client_id`:

```suggestion
table = bq_context.client.get_table(full_table_name)
return bq_context.client.list_rows(table).to_dataframe()
```

This might be a little more performant